libs/dh/message-archive/feature-search/src/lib/page.component.ts

```diff
@@ -0,0 +1,40 @@
+/**
+ * @license
+ * Copyright 2020 Energinet DataHub A/S
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License2");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { Component, signal } from '@angular/core';
+import { ArchivedMessage } from '@energinet-datahub/dh/message-archive/domain';
+
+import { DhMessageArchiveSearchDetailsComponent } from './details.component';
+import { DhMessageArchiveSearchStartComponent } from './start.component';
+import { DhMessageArchiveSearchTableComponent } from './table.component';
+
+@Component({
+  selector: 'dh-message-archive-search-page',
+  standalone: true,
+  imports: [
+    DhMessageArchiveSearchDetailsComponent,
+    DhMessageArchiveSearchStartComponent,
+    DhMessageArchiveSearchTableComponent,
+  ],
+  template: `
+    <dh-message-archive-search-details [message]="message()" (close)="message.set(undefined)" />
+    <dh-message-archive-search-start #start (start)="table.fetch($event)" (clear)="table.reset()" />
```

Comment:
suggestion: it might be worth instatiating the modal component dynamicaly only when someone clicks the Search button. You can take a look at `WattModalService`.
This will save us some initial requests assuming most users will not use the page but end up here after login.

Reply:
Good idea! Can I look into it after merge, since it is currently behind a feature flag?

Reply:
Sure. Let's get some user feedback and improve afterwards.